### DISCUSSION TITLE
How to change the default dashboard for new users?

### DISCUSSION THREAD

**Discussion by bradley-rose** [netbox-community/netbox#12908] [Q&A] [answered]
I love the addition of the customizable dashboard that was introduced with v3.5! One issue is with the RSS feed / NetBox News widget. My NetBox instance is not Internet connected, so this widget shows an error to every new user that accesses the landing page before signing in, and every new user sees this before customizing their dashboard as well. 

I'd like to change the default layout of the dashboard to not include this widget. I can see there is a [DEFAULT_DASHBOARD parameter](https://github.com/netbox-community/netbox/blob/develop/docs/configuration/default-values.md), but I don't know how to properly implement it. 

Any assistance here would be greatly appreciated!

**Comment by willikubny** [comment, marked as answer]
Hi, I've followed this description: [https://github.com/netbox-community/netbox/issues/12384](https://github.com/netbox-community/netbox/issues/12384)

I use netbox-docker and added the DEFAULT_DASHBOARD constant to configuration/extras.py

I hope that will help you.
